### PR TITLE
Prelaunch Home

### PR DESCRIPTION
**Pull request by asermax**
## What does this PR do?

This PR does another refactor of the layout on top of the one done in #16, to adapt to the new home for prelaunch design. We still need to do some work on the layout to make it flexible enough to handle the other sections (@homer0 let's talk later about this).

Other than that, added the new elements and re-ordered already existing parts of the page to conform to the [new design for the home](https://xd.adobe.com/spec/5450024d-4855-4272-41cd-a571f05c247f-2261/).

## How should it be tested manually?
Open the now deploy for this PR and compare against the design mentioned above.

### CODE REVIEW

File: src/components/AddEmailForm/AddEmailForm.jsx [hunk 1]
```diff
@@ -31,38 +31,44 @@ const _AddEmailForm = ({ emailError, isAdding, wasSaved, add }) => {
...
-          type="submit"
-          color="secondary"
-          isLoading={isAdding}
-          disabled={isAdding}
-          className={styles.button}
-        >
-          Registrarme
-        </Button>
-      </form>
+  return (
```
Comment (homer0) [inline src/components/AddEmailForm/AddEmailForm.jsx]: I refactored this whole block on #20

File: src/components/AddEmailForm/AddEmailForm.jsx [hunk 2]
```diff
@@ -31,38 +31,44 @@ const _AddEmailForm = ({ emailError, isAdding, wasSaved, add }) => {
...
+    <div className={`${styles.container} ${className ? className : ''}`}>
+      {wasSaved ? (
+        <div className={styles.success}>
+          <h3 className={styles.title}>¡Todo listo!</h3>
+          <p className={styles.text}>Pronto te estaremos escribiendo.</p>
+        </div>
+      ) : (
+        <Fragment>
```
Comment (homer0) [inline src/components/AddEmailForm/AddEmailForm.jsx]: Isn't this the same as `<> ... </>`?

File: src/layouts/default/index.jsx [hunk 3]
```diff
@@ -13,6 +13,20 @@ import SEO from 'components/seo';
 // Styles
 import styles from './styles.module.scss';
 
+export const Sidebar = ({ children, className, ...props }) => (
+  <aside className={`${styles.sidebar} ${className ? className : ''}`} {...props}>
+    {children}
+  </aside>
+);
+
+export const Content = ({ children, className, ...props }) => (
```
Comment (homer0) [inline src/layouts/default/index.jsx]: You are not using `props`.

File: src/layouts/default/index.jsx [hunk 4]
```diff
@@ -13,6 +13,20 @@ import SEO from 'components/seo';
...
+  <main className={`${styles.content} ${className ? className : ''}`}>{children}</main>
+);
+
+export const Footer = ({ children, className, ...props }) => (
```
Comment (homer0) [inline src/layouts/default/index.jsx]: You are not using `props`.

File: src/pages/index.jsx [hunk 5]
```diff
@@ -4,45 +4,17 @@ import React from 'react';
...
-      <div className={styles.bracketContainer}>
-        <img
-          className={`${styles.bracket} ${styles.bracketRight} ${styles.hiddenXs}`}
-          src={JumboBracketImage}
-          aria-hidden="true"
-          alt=""
-        />
-      </div>
-    </div>
+    <div className={styles.middleSection} />
```
Comment (homer0) [inline src/pages/index.jsx]: wa?

File: src/components/SocialNetworkLinks/SocialNetworkLinks.jsx [hunk 6]
```diff
@@ -5,28 +5,19 @@ import InstagramIcon from 'assets/icons/instagram.svg';
 import { SOCIAL } from 'data/constants';
 import styles from './styles.module.scss';
 
-export const SocialNetworkLinks = ({ useContainer }) => {
-  const _useContainer = typeof useContainer === 'undefined' ? true : useContainer;
```
Comment (homer0) [inline src/components/SocialNetworkLinks/SocialNetworkLinks.jsx]: I see what you did there xD.

File: src/components/AddEmailForm/AddEmailForm.jsx (hunk 1 above)
Comment (asermax) [reply src/components/AddEmailForm/AddEmailForm.jsx]: I'll rebase :+1:

File: src/components/AddEmailForm/AddEmailForm.jsx (hunk 2 above)
Comment (asermax) [reply src/components/AddEmailForm/AddEmailForm.jsx]: Yup, you used to need a babel plugin for this so I kept using the long version. I'll change it.

File: src/pages/index.jsx (hunk 5 above)
Comment (asermax) [reply src/pages/index.jsx]: Lol, that was the commit hook that applies prettier rules (the `div` was empty so it autoclosed it).

File: src/components/SocialNetworkLinks/SocialNetworkLinks.jsx (hunk 6 above)
Comment (asermax) [reply src/components/SocialNetworkLinks/SocialNetworkLinks.jsx]: :eyes:

**Comment by joelalejandro** [comment]
@homer0 @asermax Let's hide the menu and deploy this so we can the party started :D

**Pull request merged**